basic_distributions/wishart2.stan

```diff
@@ -1,20 +1,20 @@
 transformed data {
   cov_matrix[4] S;
 
-  S[1,1] <- 2.9983662;
-  S[1,2] <- 0.2898776;
-  S[1,3] <- -2.650523;
-  S[1,4] <- 0.1055911;
-  S[2,2] <- 11.4803610;
-  S[2,3] <- 7.157993;
-  S[2,4] <- -3.1129955;
-  S[3,3] <- 11.676181;
-  S[3,4] <- -3.5866852;
-  S[4,4] <- 1.4482736;
+  S[1,1] = 2.9983662;
+  S[1,2] = 0.2898776;
+  S[1,3] = -2.650523;
+  S[1,4] = 0.1055911;
+  S[2,2] = 11.4803610;
+  S[2,3] = 7.157993;
+  S[2,4] = -3.1129955;
+  S[3,3] = 11.676181;
+  S[3,4] = -3.5866852;
+  S[4,4] = 1.4482736;
 
   for (m in 1:4)
     for (n in 1:(m-1))
-      S[m,n] <- S[n,m];
+      S[m,n] = S[n,m];
```

Comment:
If the above was filling the lower triangular parts we could use symmetrize_from_lower_tri but no sense complicating I guess.